<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>department</class>
 <widget class="QMainWindow" name="department">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="deleteLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>680</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Department ID</string>
    </property>
   </widget>
   <widget class="QToolButton" name="back">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
	background: #39AAFF;
	color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/back.png</normaloff>:/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1621</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Departements Details</string>
     </attribute>
     <widget class="QTableView" name="departments_table">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>971</width>
        <height>361</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1211</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Search for department that you want</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>10</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLine">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Update Departement</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>401</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="idLine">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLine_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="save_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>350</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>111</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Departements Statistics</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Number of Department</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Number of Instructors</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>160</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string> Number of Episodes</string>
      </property>
     </widget>
     <widget class="QLabel" name="dep_counter">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>160</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;Century Gothic&quot;;
color: rgb(6, 0, 203);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="ins_counter">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>260</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;Century Gothic&quot;;
color: rgb(6, 0, 203);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="epi_counter">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>160</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;Century Gothic&quot;;
color: rgb(6, 0, 203);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>260</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string> Number of Subjects</string>
      </property>
     </widget>
     <widget class="QLabel" name="sub_counter">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>260</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;Century Gothic&quot;;
color: rgb(6, 0, 203);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Add Departement</string>
     </attribute>
     <widget class="QPushButton" name="add_btn_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>170</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Add New</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>110</y>
        <width>427</width>
        <height>182</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="nameLine">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>237</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="update_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>680</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 35pt &quot;Century Gothic&quot;;
color: rgb(56, 56, 191);
background-color: rgb(199, 197, 199);</string>
    </property>
    <property name="text">
     <string>Departements management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>680</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>deleteLine</zorder>
   <zorder>label_22</zorder>
   <zorder>back</zorder>
   <zorder>tabWidget</zorder>
   <zorder>update_btn</zorder>
   <zorder>delete_btn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
